pkg/jvscrypto/rotation_handler.go

```diff
@@ -221,34 +221,42 @@ func (h *RotationHandler) shouldDisable(ctx context.Context, ver *kmspb.CryptoKe
 	if shouldDisable {
 		logger.Info("version created before cutoff date, should disable.", zap.Any("version", ver), zap.Any("cutoff", cutoff))
 	} else {
-		logger.Debug("version created before cutoff date, no action necessary.", zap.Any("version", ver), zap.Any("cutoff", cutoff))
+		logger.Debug("version created after cutoff date, no action necessary.", zap.Any("version", ver), zap.Any("cutoff", cutoff))
```

Comment:
It can be a followup, but we can use `Debugw` instead of `Debug` to clean this up:

```suggestion
		logger.Debugw("version created after cutoff date, no action necessary.", "version", "cutoff", cutoff)
```

Reply:
Sure, will do this as a follow up. Do you mean we prefer Debugw than Debug and Debugf in general? If so, saw a few other places using Debug and Debugf, can clean them up together.

Reply:
I prefer Debugw because we're not really concerned with allocations.